Header header
int64 x1
int64 y1
int64 heading1
int64 tagid1
int64 x2
int64 y2
int64 heading2
int64 tagid2
